src/applications/disability-benefits/all-claims/pages/reservesNationalGuardService.jsx

```diff
@@ -22,30 +19,10 @@ export const uiSchema = {
           pattern: 'End date must be after start date',
           required: 'Please enter a date',
         },
-        from: {
-          'ui:title': 'Obligation start date',
-          'ui:widget': 'date',
-          'ui:webComponentField': VaMemorableDateField,
-          'ui:validations': [validateDate],
-          'ui:errorMessages': {
-            pattern: 'Please enter a valid date',
-            required: 'Please enter a date',
-          },
-        },
-        to: {
-          'ui:title': 'Obligation end date',
-          'ui:widget': 'date',
-          'ui:webComponentField': VaMemorableDateField,
-          'ui:validations': [validateDate],
-          'ui:errorMessages': {
-            pattern: 'Please enter a valid date',
-            required: 'Please enter a date',
-          },
-        },
-      },
-      unitName: {
-        'ui:title': 'Unit name',
+        from: { ...dateUI('Obligation start date') },
+        to: { ...dateUI('Obligation end date') },
```

Comment:
You don't really need to use the spread operator here, but I don't think it should be a blocker

```suggestion
        from: dateUI('Obligation start date'),
        to: dateUI('Obligation end date'),
```

Reply:
Made the requested modification.